CHALLENGE 2.0/def,trian_calc.py

```diff
@@ -0,0 +1,14 @@
+def trian_calc (A,B,C):
+    A=float(input("insert a number here; "))
+    B=float(input("insert a number here; "))
+    C=float(input("insert a number here; "))
+    if  A+B > C and  A+C > B and B+C > A:
+        print(True)
+    else:
+        print(False)
```

Comment:
This function does not return anything. I am expecting it to return the answer. So don't use `print`, return the answer

Reply:
done!

Reply:
You have not resolved this. Use this kind of return

```suggestion
        return False
```